### ISSUE TITLE
Implementation plan for 3D Model support

### ISSUE THREAD

**Issue by zackkrida** [WordPress/openverse#585]
We need to outline the specifics for including 3D Model support in the Openverse Frontend.
The [3D Model Support RFC](https://github.com/WordPress/openverse/blob/main/rfcs/20220221-3d_model_support.md) is a great outline of _approximately_ what needs to be done here, but delving into further detail is blocked by a few things:

- Design work to better understand any components we need to build
- 3D model data in the API, or
- Mocked API responses for 3D models

Once we have this implementation plan we'll need to create many small issues to track this work.

**Comment by sarayourfriend** [comment]
Closing as 3d models are indefinitely on hold and no work should be done on them until we re-plan this project.
